parachains/launch-a-parachain/obtain-coretime.md

```diff
@@ -1,82 +1,177 @@
+### Placing an On-Demand Order
 
-## Obtaining Coretime
+![Placing an on-demand order for coretime](/images/parachains/launch-a-parachain/obtain-coretime/obtain-coretime-1.webp)
 
-### Bulk Coretime
+To place an on-demand coretime order, follow these steps:
 
-Bulk coretime provides several advantages:
+1. Open the [Polkadot.js Apps interface connected to the Polkadot TestNet (Paseo)](https://polkadot.js.org/apps/?rpc=wss://paseo.dotters.network){target=\_blank}.
 
-- Monthly allocation of resources
-- Guaranteed block production slots (every 12 seconds, or 6 seconds with [Asynchronous Backing](https://wiki.polkadot.com/learn/learn-async-backing/#asynchronous-backing){target=\_blank})
-- Priority renewal rights
-- Protection against price fluctuations
-- Ability to split and resell unused coretime
+2. Navigate to **Developer > Extrinsics** in the top menu.
 
-To purchase bulk coretime:
+3. Select the account that registered your parachain ID.
 
-1. Access the Coretime system parachain.
-2. Interact with the Broker pallet.
-3. Purchase your desired amount of coretime.
-4. Assign the purchased core to your registered `ParaID`.
+4. From the pallet dropdown, select `onDemand` and then choose `placeOrderAllowDeath` as the extrinsic.
 
-After successfully obtaining coretime, your parachain will automatically start producing blocks at regular intervals.
+5. Configure the parameters:
+    - **maxAmount**: The maximum amount of tokens you're willing to spend (e.g., `1000000000000`). This value may vary depending on network conditions.
+    - **paraId**: Your reserved parachain ID (e.g., `4508`).
 
-For current marketplaces and pricing, consult the [Coretime Marketplaces](https://wiki.polkadot.com/learn/learn-guides-coretime-marketplaces/){target=\_blank} page on the Polkadot Wiki.
+6. Review the transaction details and click **Submit**.
 
-### On-demand Coretime
+Upon successful submission, your parachain will produce a new block. You can verify this by checking your collator node logs, which should display output confirming block production.
 
-On-demand coretime allows for flexible, as-needed block production. To purchase:
+!!!note
+    Each successful on-demand extrinsic will trigger one block production cycle. For continuous block production, you'll need to place multiple orders or consider bulk coretime.
 
-1. Ensure your collator node is fully synchronized with the relay chain.
-2. Submit the `onDemand.placeOrderAllowDeath` extrinsic on the relay chain with:
+## Purchase Bulk Coretime
 
-    - **`maxAmountFor`**: Sufficient funds for the transaction.
-    - **`paraId`**: Your registered `ParaID`.
+Bulk coretime provides a more economical option for ongoing block production. You purchase a core on the [Coretime Chain](https://wiki.polkadot.com/learn/learn-system-chains/#coretime-chain){target=\_blank} for a specified duration (up to 28 days), which must be renewed upon expiration.
```

Comment:
@brunopgalvao This kind of confuses me because it's talking about buying a core on the Coretime Chain, but then we have instructions for using RegionX marketplace. Is through the marketplace the only way to do it or just the easiest way to do it?

Reply:
Using RegionX is the easiest way to do it. Think of RegionX as a UI around the Coretime Chain. The other more manual way would be to go directly to the Coretime Chain and execute extrinsics there.